### PR TITLE
Update Stackdriver Logging details

### PR DESCRIPTION
**Pull request by charlesbaer**
Updated the Stackdriver Logging section to include the Cloud Ops for GKE resource model and guidance on how to use.
Updated the options for exports.

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
Pls sign the CLA in order for the PR to be reviewed.
more info
https://github.com/kubernetes/community/blob/master/CLA.md

**Comment by charlesbaer** [comment]
The CLA request has been completed

**Comment by charlesbaer** [comment]
> I believe we are trying to exclude vendor-specific or dual-hosted contents in the website.

The page title is "Logging Using Stackdriver" which scopes the contents of the docs page to using Stackdriver for Kubernetes logs. In this PR, I updated the incomplete and out-dated references.  As they currently stand, users may be confused when they start using Stackdriver Logging based on the contents of this document. I did not add new content, but rather, corrected or expanded the existing content.

**Comment by celestehorgan** [comment]
Preview link: https://deploy-preview-26925--kubernetes-io-master-staging.netlify.app/docs/tasks/debug-application-cluster/logging-stackdriver/#verifying-your-logging-agent-deployment

File: content/en/docs/tasks/debug-application-cluster/logging-stackdriver.md
Comment (celestehorgan) [inline content/en/docs/tasks/debug-application-cluster/logging-stackdriver.md:221]:
```suggestion
types are used even if the Kubernetes cluster is not on Google Kubernetes Engine.
```

**Review by celestehorgan — CHANGES_REQUESTED** [review]
Per the third party content section of the content guide: https://kubernetes.io/docs/contribute/style/content-guide/#third-party-content:

- I agree with @charlesbaer, this is an update of existing documentation and it is necessary for Kubernetes to function.
- While this PR _does_ introduce mention (and link to) Cloud Ops for GKE that wasn't previously present, I think it's necessary, as logging behaviour differs because of it. 

No major changes to this PR, but one typo fix.

**Comment by celestehorgan** [comment]
I'd love if someone from @kubernetes/sig-instrumentation-approvers could take a look at this and give a technical LGTM. 

/sig instrumentation

**Comment by ehashman** [comment]
@celestehorgan we removed support of stackdriver in our e2e tests, so we may want to remove this page entirely. I'll add to SIG Instrumentation agenda this week to either find a reviewer or make a decision.

**Comment by celestehorgan** [comment]
Thanks @ehashman! Loop back round here or in slack and let me know how this goes.

**Comment by ehashman** [comment]
> I believe we are trying to exclude vendor-specific or dual-hosted contents in the website.

SIG Instrumentation is fine with this (and removing this vendor-specific doc), but we think these docs don't belong to us. Pinging SIG Cloud Provider.

**Comment by tengqm** [comment]
/hold
So the current consensus is that this page should be removed rather than fixed.

**Comment by kbhawkey** [comment]
Related to PR https://github.com/kubernetes/website/pull/28166.

**Comment by kbhawkey** [comment]
👋 @charlesbaer .
Since there has been no recent activity on this PR, I plan to close it.
Feel free to reopen this pull request.
/close

**Pull request closed without merging**
